src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -551,19 +560,18 @@ public static ConfigDef getConfig() {
     );
 
     visible.define(
-        "topics.dir",
+        TOPICS_DIR_CONFIG,
         Type.STRING,
-        "topics",
+        TOPICS_DIR_DEFAULT,
         Importance.HIGH,
-        "Top level directory to store the data ingested from Kafka. "
-            + "Supports ``${topic}`` in the value, which will be replaced by the actual"
-            + " topic name. Supports ``${1}``, ..., ``${n}`` in conjunction with "
-            + TOPIC_REGEX_CAPTURE_GROUP_CONFIG + ". See " + TOPIC_REGEX_CAPTURE_GROUP_CONFIG
-            + " configuration documentation for details.",
+        TOPICS_DIR_DOC + " Supports ``${topic}`` in the value, which will be "
```

Comment:
Why this doc string is not a constant like the rest?

Reply:
fixed